Sources/Components/Feedback/FeedbackView.swift

```diff
@@ -319,6 +317,7 @@ private class ButtonView: UIView {
             positiveButton.removeFromSuperview()
             negativeButton.removeFromSuperview()
         }
+        negativeButton.style = Button.Style.negativeButton()
```

Comment:
you can do `let button = Button(style: .negativeButton(), size: .small)` in the lazy var instead